grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -46,12 +49,22 @@ class ClientJobStatus {
         this.jcrNodesWritten = jcrNodesWritten
     }
 
-
-    public static ClientJobStatus get(@Nonnull JobExplorer explorer, @Nonnull Long jobExecutionId) {
+    /**
+     * Queries the status of a job
+     * @param explorer job explorer to query
+     * @param jobExecutionId the ID to search
+     * @throws {@link NonExistentJobException} if job can not be found. {@link IllegalArgumentException} if explorer or jobExecutionId are null
```

Comment:
You can have multiple `@throws` declarations, and the standard javadoc/groovydoc processor assumes the first "word" is the class of the exception type, so it's automatically linked.

```groovy
/**
 * @throws NonExistentJobException if job can not be found
 * @throws IllegalArgumentException if explorer or jobExecutionId are null
 */
```